src/js/trajectory_alignment.js

```diff
@@ -626,48 +618,23 @@ export class trajectory_alignment {
 			}
 			return legendOrder;
 		}
-		// TODO: refactor to shorter
 		function direction(tech) {
 			switch (tech) {
 				case 'Oil':
-					return true;
-					break;
 				case 'Coal':
-					return true;
-					break;
 				case 'Gas':
-					return true;
-					break;
-				case 'Electric':
-					return false;
-					break;
-				case 'Hybrid':
-					return false;
-					break;
 				case 'ICE':
-					return true;
-					break;
 				case 'CoalCap':
-					return true;
-					break;
 				case 'GasCap':
+				case 'OilCap':
 					return true;
-					break;
+				case 'Electric':
+				case 'Hybrid':
 				case 'HydroCap':
-					return false;
-					break;
 				case 'NuclearCap':
-					return false;
-					break;
-				case 'OilCap':
-					return true;
-					break;
 				case 'RenewablesCap':
-					return false;
-					break;
 				case 'FuelCell':
 					return false;
-					break;
 				default:
 					console.log('undefined tech:', tech);
```

Comment:
does this actually work as expected? e.g. if the value is `Oil`, won't it now skip all other cases that it doesn't match and end up doing what's defined in `default` rather than returning `true`?

Reply:
Posting a screenshot of the dashboard as built on `main` to show that directions appear as expected: 

Oil, decreases as expected:
<img width="309" alt="Screenshot 2024-12-02 at 18 51 34" src="https://github.com/user-attachments/assets/fd61f774-7b49-4a5e-8dd2-881120caf090">

Counter-example
Hydro-cap, increases as expected:
<img width="283" alt="Screenshot 2024-12-02 at 18 52 17" src="https://github.com/user-attachments/assets/5205d5f3-a8d5-4386-b5cc-a86cbfe915aa">

Reply:
Although... it's also possible that this function just isn't being called at all lol:
https://github.com/search?q=repo%3ARMI-PACTA%2Fpacta-dashboard-svelte%20direction&type=code

Reply:
🤷🏻 very strange syntax

Reply:
Search for "fallthrough" here to see the behaviour documented (in case your interested): 
https://www.w3schools.com/jsref/jsref_switch.asp

Reply:
But indeed, I don't disagree